saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EmailSpam/SpamApp.scala

```diff
@@ -14,8 +14,8 @@ import scala.collection.JavaConversions._
 
 object SpamApp extends Logging {
 
-  val trainData = new DocumentReader("../data/EmailSpam/train").docs.toList
-  val testData = new DocumentReader("../data/EmailSpam/test").docs.toList
+  val trainData = new DocumentReader("data/EmailSpam/train").docs.toList
```

Comment:
Had to change this to make the web-app work seamlessly without any code change. Need to figure out how to unify paths for `sbt` and web-app later.